README.md

```diff
@@ -1,142 +1,88 @@
-```
-
-`--unsafe-perm` is only required on systems where the global install directory is not user-writable.
-This allows npm install steps to download and save prebuilt native binaries. You may be able to omit it,
-especially if you're using a user-managed node install such as [nvm](https://github.com/creationix/nvm).
-
-### Additional Dependencies
-
-* The `balena ssh` command requires a recent version of the `ssh` command-line tool to be available:
-  * MacOS and Linux usually already have it installed. Otherwise, search for the available
-  packages on your specific Linux distribution, or for the Mac consider the [Xcode command-line
-  tools](https://developer.apple.com/xcode/features/) or [homebrew](https://brew.sh/).
-
-  * Microsoft released an OpenSSH version of ssh for Windows 10, which we understand is
-  automatically installed through Windows Update, but can be manually installed too.
-  More information [here](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse). For other versions of Windows, there are several ssh/OpenSSH clients
-  provided by 3rd parties.
-
-  * If you need `ssh` to work behind a proxy, you also need [`proxytunnel`](http://proxytunnel.sourceforge.net/) installed (available as `proxytunnel` package for Ubuntu, for example).
-
-* The `balena sync` command currently requires `rsync` (>= 2.6.9) to be installed:
-  * Linux: `apt-get install rsync`
-  * MacOS: [Xcode command-line tools](https://developer.apple.com/xcode/features/) or [homebrew](https://brew.sh/)
-  * Windows: use MinGW as described in the Windows Support section.
-
-### Windows Support
-
-We aim at supporting the standard Windows Command Prompt (`cmd.exe`) and the Windows [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/getting-started-with-windows-powershell?view=powershell-6).
-
-Some CLI commands like `balena sync` and `balena ssh` have not been thoroughly tested with the standard Windows shells. We are aware of users having a good experience with alternative shells, including:
-
-* Microsoft's [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about) (a.k.a. Microsoft's "bash for Windows 10").
+* Microsoft's [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about)
+  (a.k.a. Microsoft's "bash for Windows 10").
 * [Git for Windows](https://git-for-windows.github.io/).
-* [MinGW](http://www.mingw.org)
-  1. Install [MinGW](http://www.mingw.org).
-  2. Install the `msys-rsync` and `msys-openssh` packages.
-  3. Add MinGW to the `%PATH%` if this hasn't been done by the installer already. The location where the binaries are places is usually `C:\MinGW\msys\1.0\bin`, but it can vary if you selected a different location in the installer.
-  4. Copy your SSH keys to `%homedrive%%homepath\.ssh`.
-  5. If you need `ssh` to work behind the proxy you also need to install [proxytunnel](http://proxytunnel.sourceforge.net/)
```

Comment:
I've deleted MinGW steps 3, 4 and 5 above because:
* MinGW's own docs already cover the bit about the PATH, and the balena CLI also covers how to add `balena` to the PATH.
* The balena CLI itself does not require ssh keys being added, does it? It is 'git push' and standard ssh that may require it. If we were to provide instructions on ssh keys, they should be added for all platforms, not just MinGW.
* The proxytunnel bit is covered in INSTALL.md, in the context of `balena ssh`.
* MinGW is only one of 5 shell options now mentioned for Windows.

Reply:
> _The balena CLI itself does not require ssh keys being added, does it?_

Answering my own question, actually the `balena ssh` command seems to require a ssh key to be added to the user's homedir -- I assume the same key(s) that are used for the `git push` operation. I tested this on a fresh Windows 10 VM. I have added a section "Configuring SSH Keys" to the `INSTALL.md` file. As we are now saying that the supported shells are the `cmd.exe` command prompt or PowerShell, I've used `%userprofile%\.ssh\id_rsa.pub` as the location of the public key. I tested it successfully with the "built-in" Windows 10 ssh tool, and with `balena ssh`.